aries_cloudagent/messaging/jsonld/credential.py

```diff
@@ -93,20 +100,22 @@ async def jws_verify(verify_data, signature, public_key, wallet):
     return verified
 
 
-async def sign_credential(credential, signature_options, verkey, wallet):
+async def sign_credential(credential, signature_options, verkey, session):
```

Comment:
To be consistent with other functions in ACA-Py, I recommend we rearrange these parameters so that session comes first.

Reply:
I will fix this.